.spelling

```diff
@@ -565,105 +565,3 @@ md#renew
-# so that the x509 spelling is accepted in those files.
-
- - content/v0.12-docs/concepts/certificate.md
-x509
- - content/v0.12-docs/concepts/certificaterequest.md
-x509
- - content/v0.12-docs/release-notes/release-notes-0.5.md
-x509
- - content/v0.12-docs/release-notes/release-notes-0.6.md
-x509
- - content/v0.12-docs/release-notes/release-notes-0.7.md
-x509
- - content/v0.12-docs/release-notes/release-notes-0.9.md
-x509
- - content/v0.12-docs/tutorials/venafi/venafi.md
-x509
- - content/v0.12-docs/usage/certificate.md
-x509
- - content/v0.13-docs/concepts/certificate.md
-x509
- - content/v0.13-docs/concepts/certificaterequest.md
-x509
- - content/v0.13-docs/release-notes/release-notes-0.13.md
-x509
- - content/v0.13-docs/release-notes/release-notes-0.5.md
-x509
- - content/v0.13-docs/release-notes/release-notes-0.6.md
-x509
- - content/v0.13-docs/release-notes/release-notes-0.7.md
-x509
- - content/v0.13-docs/release-notes/release-notes-0.9.md
-x509
- - content/v0.13-docs/tutorials/venafi/venafi.md
-x509
- - content/v0.13-docs/usage/certificate.md
-x509
- - content/v0.14-docs/concepts/certificate.md
-x509
- - content/v0.14-docs/concepts/certificaterequest.md
-x509
- - content/v0.14-docs/release-notes/release-notes-0.13.md
-x509
- - content/v0.14-docs/release-notes/release-notes-0.5.md
-x509
- - content/v0.14-docs/release-notes/release-notes-0.6.md
-x509
- - content/v0.14-docs/release-notes/release-notes-0.7.md
-x509
- - content/v0.14-docs/release-notes/release-notes-0.9.md
-x509
- - content/v0.14-docs/tutorials/venafi/venafi.md
-x509
- - content/v0.14-docs/usage/certificate.md
-x509
- - content/v0.15-docs/concepts/certificate.md
-x509
- - content/v0.15-docs/concepts/certificaterequest.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.13.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.15.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.5.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.6.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.7.md
-x509
- - content/v0.15-docs/release-notes/release-notes-0.9.md
-x509
- - content/v0.15-docs/tutorials/venafi/venafi.md
-x509
- - content/v0.15-docs/usage/certificate.md
-x509
- - content/v0.16-docs/concepts/certificate.md
-x509
- - content/v0.16-docs/concepts/certificaterequest.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.13.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.15.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.16.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.5.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.6.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.7.md
-x509
- - content/v0.16-docs/release-notes/release-notes-0.9.md
-x509
- - content/v0.16-docs/tutorials/venafi/venafi.md
-x509
- - content/v0.16-docs/usage/certificate.md
-x509
- - content/v0.16-docs/usage/kubectl-plugin.md
-x509
```

Comment:
Upon removing these, the spell checker found some other `x509` references in the 1.0 and 1.1 docs, so I'm not convinced that these file specific exceptions were working as expected.

I've fixed those 1.0 and 1.1 references.